Private match

eduardomario opened the pull request.

cclauwers committed:
feature PrivateMatch v1

cclauwers committed:
feature PrivateMatch modified. v2

cclauwers committed:
Feature and StepDefs modified

cclauwers committed:
PrivateMatchRepository added

cclauwers committed:
WebSecurityConfig modified, privatematches added

cclauwers committed:
Merge branch 'master' into PrivateMatch

# Conflicts:
#	src/main/java/cat/udl/eps/softarch/mypadel/config/WebSecurityConfig.java

cclauwers committed:
joinPrivateMatch feature and stepdefs

cclauwers committed:
ReservationStepDefs

cclauwers committed:
joinPrivateMatchSteps solved

bergacat1 commented on src/main/java/cat/udl/eps/softarch/mypadel/handler/joinPrivateMatchEventHandler.java:
```diff
@@ -0,0 +1,4 @@
+package cat.udl.eps.softarch.mypadel.handler;
+
+public class joinPrivateMatchEventHandler {
```
What is this handler?

bergacat1 commented on src/test/resources/features/JoinPrivateMatch.feature:
```diff
@@ -0,0 +1,22 @@
+			Then I successfully joined a private match
+
+
+	Scenario: An invited player joins a private match
```
The format of this scenario is not the correct one. It should be something like: 
Given I login as ...
And There is a private match for...
And The player didn't receive an invitation...
When I join the private match
Then The status is 401

cclauwers committed:
Handler Eliminated
Modified in JoinPrivateMatchStepsDefs
Modified Scenarios in JoinPrivateMatch.feature

cclauwers committed:
Merge branch 'master' into PrivateMatch

# Conflicts:
#	src/main/java/cat/udl/eps/softarch/mypadel/config/WebSecurityConfig.java
#	src/main/java/cat/udl/eps/softarch/mypadel/repository/PrivateMatchRepository.java

cclauwers committed:
PrivateMatchFeature Admins Scenario Changed

cclauwers committed:
Merge branch 'master' into PrivateMatch
